auth/tasks.py

```diff
@@ -50,51 +51,57 @@ def update_group_and_perms(department, created):
         if created:
             assign_model_perms_for_department(department)
 
+    try:
+        for raw_department in raw_departments:
+            department, created = Department.objects.get_or_create(
+                raw_department_id=raw_department.dwid,
+                defaults={'name': raw_department.dwmc})
+            update_group_and_perms(department, created)
+            updated = False
+            # 同步隶属单位
+            if (department.super_department is None or
+                    department.super_department.raw_department_id !=
+                    raw_department.lsdw):
+                super_department_name = DepartmentInformation.objects.get(
+                    dwid=raw_department.lsdw).dwmc
+                super_department, created = Department.objects.get_or_create(
+                    raw_department_id=raw_department.lsdw,
+                    defaults={'name': super_department_name}
+                )
+                update_group_and_perms(super_department, created)
+                department.super_department = super_department
```

Comment:
When `super_department` of the `department` changed, all users belonging to this `department` should be moved out from the groups of `super_department` and its ancestors.